### PR TITLE
Niko/filter non english

### PR DESCRIPTION
**Pull request by niko79542**
Regex for Unicode characters had some false positives. Out of a sample of 70 posts...the English ones had between a 0% to 2% occurrence of Unicode characters. Out of 70, there were 3 non-English posts. These had 30%, 65% and 65% Unicode characters in the post body.

This PR marks posts with 10% occurrence or less as English. Think this gives us a wide margin.

### CODE REVIEW

File: src/controllers/graphql/resolvers/post/postResolver.js [hunk 1]
```diff
@@ -21,14 +21,21 @@ module.exports = {
     },
     getHundredxPosts: async (_, args) => {
       return await PostService.fetchHundredxResteemedPosts();
-    }
+    },
+  },
+  Mutation: {
+    broadcastPost: async (_, { permLink, title, body, tags }, { authenticatedUserInstance } ) => {
+      return !authenticatedUserInstance
```
Comment (jay-ithiel) [inline src/controllers/graphql/resolvers/post/postResolver.js]: Let's use an `if` statement here
```
broadcastPost: async() => {
   if (!authenticatedUserInstance) {
      return new AuthenticationError('');
   }
   await PostService.callFunction();
}
```

Comment (jay-ithiel) [reply src/controllers/graphql/resolvers/post/postResolver.js]: Ternaries should be used for very simple operations. A general rule of thumb is if you have to break apart your ternary into multiple lines, you should probably be using an `if` statement

Comment (jay-ithiel) [reply src/controllers/graphql/resolvers/post/postResolver.js]: We're also using a ternary for error checking in `getFollowerPosts`. Could we change that to an `if statement as well?

File: src/services/postService.js [hunk 2]
```diff
@@ -18,23 +18,40 @@ const {
 const VIDEO_URLS   = require('../utils/videoUrls');
 
 module.exports = {
+  broadcastAndStorePost: function({ authenticatedUserInstance, permLink, title, body, tags }) {
+    return authenticatedUserInstance.broadcastPost({ permLink, title, body, tags })
+      .then(broadcastSuccess => {
+        return broadcastSuccess
```
Comment (jay-ithiel) [inline src/services/postService.js]: This ternary is simple enough to be read on 1 line 👍

File: src/controllers/graphql/resolvers/post/postResolver.js (hunk 1 above)
Comment (niko79542) [reply src/controllers/graphql/resolvers/post/postResolver.js]: sure

File: src/services/postService.js (hunk 2 above)
Comment (niko79542) [reply src/services/postService.js]: cool, will change to a 1 liner

**Pull request merged**